tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/endpoints/EndpointTestParamsDeserializer.java

```diff
@@ -57,7 +60,24 @@ public static EndpointTests.EndpointTestParameter deserializeParameter(String pa
                 // this will always be ["*"], however this should not be relied upon.
                 parameter.setType(EndpointTests.EndpointTestParameter.ParameterType.STRING);
                 parameter.setStrValue("*");
-            } else {
+            }
+            else if(element.isJsonArray())
+            {
+                JsonArray jsonArray = element.getAsJsonArray();
+                parameter.setType(EndpointTests.EndpointTestParameter.ParameterType.STRING_ARRAY);
+                
+                List<String> stringArray = StreamSupport.stream(jsonArray.spliterator(), false)
+                .map(arrElem -> {
```

Comment:
please indent, this isnt proper formatting for [streams](https://cr.openjdk.org/~alundblad/styleguide/index-v6.html#toc-wrapping-lines)